pkg/awsutils/awsutils.go

```diff
@@ -44,16 +42,16 @@ import (
 
 const (
 	metadataMACPath      = "network/interfaces/macs/"
-	metadataAZ           = "placement/availability-zone/"
+	metadataAZ           = "placement/availability-zone"
 	metadataLocalIP      = "local-ipv4"
 	metadataInstanceID   = "instance-id"
 	metadataInstanceType = "instance-type"
 	metadataMAC          = "mac"
-	metadataSGs          = "/security-group-ids/"
-	metadataSubnetID     = "/subnet-id/"
-	metadataVPCcidrs     = "/vpc-ipv4-cidr-blocks/"
-	metadataDeviceNum    = "/device-number/"
-	metadataInterface    = "/interface-id/"
+	metadataSGs          = "/security-group-ids"
+	metadataSubnetID     = "/subnet-id"
+	metadataVPCcidrs     = "/vpc-ipv4-cidr-blocks"
+	metadataDeviceNum    = "/device-number"
+	metadataInterface    = "/interface-id"
```

Comment:
I guess we don't use any of these any more, since they are all hardcoded in `imds.go`?

Reply:
Right.  A bunch of them are still used in test code, so I should at least move them there.  (I was sort of hoping that unused code tool I used would tell me this, but it seems it doesn't see `const`s)